xbmc/guilib/GraphicContext.cpp

```diff
@@ -983,16 +984,17 @@ void CGraphicContext::SetMediaDir(const std::string &strMediaDir)
   m_strMediaDir = strMediaDir;
 }
 
+
+void CGraphicContext::SetStereoMode(RENDER_STEREO_MODE mode)
+{
+  m_stereoMode = mode;
+  g_application.m_pPlayer->TriggerUpdateResolution();
```

Comment:
Where could this be done?
Current master does the stereo switch from Flip and the render config switch from FrameMove which makes it hard to avoid a double hdmi mode change.